docs/src/data/benchmarks.md

```diff
@@ -70,7 +70,7 @@ The benchmark run tested DiceDB with 24 threads, 50 client connections per threa
   - p99 latency was 8.89 ms, indicating that 99% of requests were faster than this.
   - p99.9 latency reached 17.91 ms, showing how the highest-latency outliers behaved.
 
-## Redis Benchmark
+## DiceDB Benchmark
```

Comment:
DiceDB benchmark are already listed as part of L20. We need to update below script on how to build redis and run it so that `memtier_benchmark` can be run.